layerindex/tasks.py

```diff
@@ -14,11 +26,50 @@
 
 tasks = Celery('layerindex',
     broker=settings.RABBIT_BROKER,
-    backend=settings.RABBIT_BACKEND)
+    backend=settings.RABBIT_BACKEND,
+    broker_heartbeat=0)
 
 @tasks.task
 def send_email(subject, text_content, from_email=settings.DEFAULT_FROM_EMAIL, to_emails=[]):
     # We seem to need to run this within the task
     utils.setup_django()
     msg = EmailMessage(subject, text_content, from_email, to_emails)
     msg.send()
+
+@tasks.task(bind=True)
+def run_update_command(self, branch_name, update_command):
+    utils.setup_django()
+    from layerindex.models import Update
+    updateobj = Update.objects.get(task_id=self.request.id)
+    updateobj.started = datetime.now()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The naive datetime objects are treated by many datetime methods as local times, it is preferred to use aware datetimes to represent times in UTC. The recommended way to create an aware datetime object representing a specific timestamp in UTC is by passing `tzinfo` as an argument to the method.

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)

Similar issue at line number 72.